pkg/cmd/pr/diff/diff.go

```diff
@@ -90,6 +94,10 @@ func diffRun(opts *DiffOptions) error {
 		return err
 	}
 
+	if pager, err := getDefaultPager(); err == nil {
```

Comment:
This code looks ok by itself, but we should also take precaution to _never_ use a pager if the output has been redirected. You can use `opts.IO.IsStdoutTTY()` to check that

Reply:
✅ Checking redirections before using pager
✅ Added test